environment_linux.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - ffmpeg
   - gprof2dot
   - graphviz
+  - h5py
```

Comment:
Could you add it to `environment_py3.yml` as well?

Reply:
Done, thanks!